frontend/src/app/shared/box/box.component.ts

```diff
@@ -37,6 +37,7 @@ export class BoxTitleDirective {
 export class BoxComponent implements AfterContentInit {
   @Input() boxTitle = '';
   titleTemplateRef: TemplateRef<void> | null = null;
+  imagePath = '';
```

Comment:
Use a non-mandatory  input instead of checking the title of the box to know what to display. It will be much more modular this way